packages/react/src/components/form/Field/fieldA11Y.ts

```diff
@@ -0,0 +1,85 @@
+export const fieldA11Y = (fieldElement: HTMLElement | null) => {
```

Comment:
I'd like if we can find a different name for this.
Maybe `fieldObserver` or `fieldUtil`?
Should we also make this a named function?